### PR TITLE
Convert to date and Duplicate right clicks enhancement

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -127,6 +133,19 @@ Public Class dlgMakeDate
...
+        strSelectedColumn = strColumn
+        strSelectedDataFrame = strDataFrame
+        bUseSelectedColumn = True
+    End Sub
+
+
+    Private Sub SetDefaultColumn()
+        ucrSelectorMakeDate.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = strSelectedDataFrame
+        ucrReceiverForDate.Add(strSelectedColumn, strSelectedDataFrame)
+        bUseSelectedColumn = False
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: This looks really great. In here you also need to make sure the correct radio button is checked at the top.

Comment (stevenndungu) [inline instat/dlgMakeDate.vb]: Thanks. Should I still have the  Single Column radio checked on the sub? ...Its the default on the dialogue

**Comment by stevenndungu** [comment]
You are right. I get you right

**Comment by dannyparsons** [comment]
Great, can someone review this by testing?

**Pull request merged**